commercetools-internal-processors/src/main/java/io/sphere/sdk/annotations/processors/generators/DraftBuilderGenerator.java

```diff
@@ -128,6 +130,117 @@ private void createCopyFactoryMethods(final TypeElement resourceDraftValueType,
+        addBuilderMethodReturn(resourceDraftValueType,builder);
+        return builder.build();
+    }
+
+    private List<MethodSpec> createPlusTolListElementsMethodSpec(final TypeElement resourceDraftValueType, final List<PropertyGenModel> propertyGenModels){
+
+        return propertyGenModels.stream()
+                .filter(this::isListProperty)
+                .map(propertyGenModel -> createPlusToListElementMethodSpec(resourceDraftValueType,propertyGenModel))
+                .collect(Collectors.toList());
+
+    }
+
+    private MethodSpec createPlusToListElementMethodSpec(final TypeElement resourceDraftValueType, PropertyGenModel property){
+
+        final String methodName = "plus" + StringUtils.capitalize(property.getName());
+        final TypeName parameterTypeName = property.getType() instanceof ParameterizedTypeName ? ((ParameterizedTypeName)property.getType()).typeArguments.get(0) : ClassName.get(Object.class);
+        MethodSpec.Builder builder = MethodSpec
+                .methodBuilder(methodName)
+                .addModifiers(Modifier.PUBLIC)
+                .addJavadoc("Adds {@code $L} parameter to the {@code $L} List property of this builder.\n\n", property.getName(), property.getName())
+                .addJavadoc("@param $L the value of the element to add to $L\n", property.getJavaIdentifier(), property.getJavadocLinkTag())
+                .addJavadoc("@return this builder\n")
+                .addParameter(createParameter(property,parameterTypeName,false))
+                .addCode("this.$L =  $T.listOf($T.ofNullable(this.$L).orElseGet($T::new), $T.singletonList($L));\n", property.getName(), SphereInternalUtils.class,Optional.class, property.getName(), ArrayList.class,Collections.class,property.getName());
+        addBuilderMethodReturn(resourceDraftValueType,builder);
+        return builder.build();
+    }
+
+    private boolean isSetProperty(PropertyGenModel propertyGenModel){
+        final boolean isParmetrizedSet = (propertyGenModel.getType() instanceof ParameterizedTypeName) && ((ParameterizedTypeName)propertyGenModel.getType()).rawType.reflectionName().equals(Set.class.getTypeName());
+        final boolean isNonParmetrizedSet = (propertyGenModel.getType() instanceof ClassName) && ((ClassName)propertyGenModel.getType()).reflectionName().equals(Set.class.getTypeName());
+
+        return isNonParmetrizedSet || isParmetrizedSet;
+    }
+
+    private List<MethodSpec> createPlusSetsMethodSpec(final TypeElement resourceDraftValueType, final List<PropertyGenModel> propertyGenModels){
+
+        return propertyGenModels.stream()
+                .filter(this::isSetProperty)
+                .map(propertyGenModel -> createPlusSetMethodSpec(resourceDraftValueType,propertyGenModel))
+                .collect(Collectors.toList());
+
+    }
+
+    private MethodSpec createPlusSetMethodSpec(final TypeElement resourceDraftValueType, PropertyGenModel property){
```

Comment:
The `final` modifier is missing on the last parameter.

Reply:
good catch 👍